src/SevenWonders.js

```diff
@@ -586,6 +598,11 @@ const Turn = function(player, game, hands, index, free) {
           game.endOfRoundPayments.splice(i, 1);
           player.built.pop();
         });
+      } else if (player.freeBuildTypes.indexOf(card.type) != -1) {
+        player.built.push(card);
```

Comment:
Shouldn't this be handled by the verify() condition above?

Reply:
Right